crates/rooch-framework/sources/transaction_fee.move

```diff
@@ -2,19 +2,21 @@ module rooch_framework::transaction_fee {
 
     use moveos_std::account_storage;
     use moveos_std::context::Context;
-    use rooch_framework::coin::{Self, Coin};
+    use moveos_std::object_ref::{Self, ObjectRef};
+    use rooch_framework::coin::{Self, Coin, CoinStore};
     use rooch_framework::gas_coin::{GasCoin};
 
     friend rooch_framework::genesis;
     friend rooch_framework::transaction_validator;
 
     struct TransactionFeePool has key,store {
-        fee: Coin<GasCoin>,
+        fee: ObjectRef<CoinStore>,
```

Comment:
After refactoring according to ObjectRef, ObjectRef<CoinStore> must be passed when using Coin, which means that the specific Coin type is only known after instantiation. This place feels a little strange.

Reply:
That is because I remove <T> from CoinStore. If we want to keep the CoinStore<T>, the ObjectRef of CoinStore will be `ObjectRef<CoinStore<T>>`.

I found that erasing the CoinType can make the storage simpler.